src/app/[locale]/(profile)/_components/profile-form.tsx

```
@@ -0,0 +1,184 @@
+"use client";
+
+import Image from "next/image";
+import { useState, useEffect } from "react";
+// import { updateUserProfile, deleteAccount } from "./actions";
+import { useRouter } from "next/navigation";
+
+export default function ProfileForm() {
+  const router = useRouter();
```

ESLint

[error] 'router' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)